examples/complete/fixtures.us-east-2.tfvars

```diff
@@ -20,6 +20,6 @@ min_size = 2
 
 disk_size = 20
 
-kubeconfig_path = "/.kube/config"
+kubeconfig_path = "${HOME}/.kube/config"
```

Comment:
these variables are for the automated test.
`/.kube/config` is ok since we are running it on CodeFresh container

Reply:
I do run it on my Jenkins. So it would be nice to add ${HOME} so even on container it will use / as the home dir of root user on Docker or will use /var/lib/jenkins on Jenkins so it will be multiple CI/CD compliant if you add ${HOME}

Reply:
@omerfsen thanks
Note that this file is our example/test (variables for the test) which we deploy automatically on AS using Terratest, so it's not related to the module and how you run it.
Please don't change it in this PR so we don't mix different things in one PR.